persist/fs/commitlog/types.go

```diff
@@ -99,6 +99,9 @@ type Series struct {
 	// ID is the series identifier
 	ID ident.ID
 
+	// Tags are the series tags
+	Tags ident.Tags // FOLLOWUP(prateek): wire Tags to commit log writer
```

Comment:
We should sync on this as well because this will probably involve changing the commit log format

Reply:
Or can you just throw it in the "annotations" section?

Reply:
Talked offline: going to change the format. Will sync again when we reach the point of making this change.